src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -116,6 +116,22 @@ public static class CLIOptions
 
         };
 
+        public static readonly CLIOption<string[]> DashboardCompareFileExcludePatterns = new CLIOption<string[]>
+        {
+            ArgumentName = "--dashboard-compare-excluded-non-source-files", // TODO: Really needs a better name
```

Comment:
Any ideas for a better argument name?

Reply:
For starters this feature shouldn't only work for the dashboard compare but also the regular diff feature.

How about `--diff-ignore-files`
And allow this to be used to also ignore changes in source files.